content/software-security/cves/cve-why-care.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Why Care About Software Vulnerabilities?"
+description: "An overview of software vulnerability impacts and the significance of CVEs in vulnerability management practices"
+lead: "An overview of software vulnerability impacts and the significance of CVEs in vulnerability management practices"
+type: "article"
+date: 2023-07-13T19:46:58+00:00
+lastmod: 2023-07-13T19:46:58+00:00
+contributors: ["Michelle McAveety"]
+draft: false
+tags: ["Overview"]
+images: []
+menu:
+  docs:
+    parent: "cves"
+weight: 002
+toc: true
+---
+
+Software products are prone to [*vulnerabilities*](/software-security/cves/cve-intro/) which, if exploited by an attacker, may negatively impact the systems and consumers relying on them. Attacks against vulnerable software systems can result in the unintended exposure and misuse of sensitive data (like the theft of user account credentials). In other cases, these attacks could affect the provision of a service or compromise critical infrastructure that relies on the software. Given the considerable threat that they can pose, it is important that developers spend time mitigating vulnerabilities to protect against hackers seeking to exploit them.
+
+Addressing the vulnerabilities present in your software helps secure the systems you support, use, and maintain. In this article, you will explore why you should care about vulnerabilities as a software developer, and learn about federal regulations that draw importance to CVE management (such as FedRAMP).
+
+
+## As a Developer
+
+Discovering and mitigating software vulnerabilities is a difficult – but necessary – task for developers to tackle. Development teams are ultimately the authorities which are able to remediate vulnerabilities at the source. Depending on the severity of a vulnerability, its exploitation could significantly impact the integrity of the software product, as well as the trust, safety, and operations of the consumers who rely on it.
```

Comment:
A few "it"s in here, so I suggest being more concrete. You may want to break up that second sentence as an alternative approach.

```suggestion
Discovering and mitigating software vulnerabilities is a difficult – but necessary – task for developers to tackle. Development teams are ultimately the authorities who are able to remediate vulnerabilities at the source. Depending on the severity of a vulnerability, its exploitation could significantly impact the integrity of the software product, as well as the trust, safety, and operations of the consumers who rely on that software.
```